staging/src/k8s.io/apiserver/pkg/storage/watch_cache.go

```diff
@@ -203,21 +208,13 @@ func objectToVersionedRuntimeObject(obj interface{}) (runtime.Object, uint64, er
 	if err != nil {
 		return nil, 0, err
 	}
-	resourceVersion, err := parseResourceVersion(meta.GetResourceVersion())
+	resourceVersion, err := w.versioner.ParseListResourceVersion(meta.GetResourceVersion())
 	if err != nil {
 		return nil, 0, err
 	}
 	return object, resourceVersion, nil
 }
 
-func parseResourceVersion(resourceVersion string) (uint64, error) {
```

Comment:
:+1: good that this goes away. It assumes a resourceVersion syntax which is coincidently an uint for our storages.